src/skmultiflow/meta/multi_output_learner.py

```diff
@@ -40,7 +40,8 @@ class MultiOutputLearner(BaseSKMObject, ClassifierMixin, MetaEstimatorMixin, Mul
     >>> from skmultiflow.data.file_stream import FileStream
     >>> from sklearn.linear_model import Perceptron
     >>> # Setup the file stream
-    >>> stream = FileStream("skmultiflow/data/datasets/music.csv", 0, 6)
+    >>> stream = FileStream("https://raw.githubusercontent.com/scikit-multiflow/"
+    ...                     "master/moving_squares.csv", 0, 6)
```

Comment:
The repository does not appear in the link.

Reply:
Fixed in e8b8f3fd3bb60bd1e6ad657e3fe45da11ac0a8d2